riotfile.py

```diff
@@ -239,19 +248,21 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         ),
         Venv(
             name="tracer",
-            command="pytest {cmdargs} tests/tracer/",
+            pkgs={
+                "msgpack": latest,
+                "attrs": ["==20.1.0", latest],
+                "packaging": ["==17.1", latest],
+                "structlog": latest,
+                # httpretty v1.0 drops python 2.7 support
+                "httpretty": "==0.9.7",
+            },
             venvs=[
+                # Riot venvs break with Py 3.11 importlib, specifically with hypothesis (test_http.py).
```

Comment:
```suggestion
                # Riot venvs break with Py 3.11 importlib, specifically with hypothesis (test_http.py). We run the test_http.py tests with tox in CI.
```

can we also move this line down to above the command line